cmd/zsysd/completion.go

```diff
@@ -11,18 +12,58 @@ import (
 func installCompletionCmd(rootCmd *cobra.Command) {
 	prog := rootCmd.Name()
 	var completionCmd = &cobra.Command{
-		Use:   "completion",
-		Short: i18n.G("Generates bash completion scripts"),
-		Long: fmt.Sprintf(i18n.G(`To load completion run
+		Use:   "completion [bash|zsh|powershell]",
+		Short: i18n.G("Generates completion scripts"),
+		Long: strings.ReplaceAll(i18n.G(`To load completions:
```

Comment:
Please use fmt.Sprintf rather than ReplaceAll here, this is more familiar to translators and less up for breakage.

Reply:
I used `strings.ReplaceAll` because of the numerous locations the program name appears in the string, instead of manually counting the number of %s instances and providing duplicate parameters at the end.
If you feel that's the wrong approach, I'll switch back to `fmt.Sprintf`

Reply:
Yeah. I got why you were doing this and indeed, named replacement (like in python), would be good. But it’s still better I think to use the standard `fmt.Sprintf` to keep %s for the translators.
OR, IIRC, this is the only string replaced, correct? So you can `strings.ReplaceAll` all `%s` with `prog`? That way, we keep %s for translators without worrying about the underlined mechanic?

Reply:
OK, I'll just change the #prog# to %s, np.